live_site.py

```diff
@@ -0,0 +1,433 @@
+import os
+import json
+import click
+import requests
+import colorama
+import digitalocean
+from colorama import Fore
+from colorama import init
+from pssh.clients import SSHClient
+
+
+if os.getenv('DIGITAL_OCEAN_AUTH_TOKEN'):
+	api_token = os.getenv('DIGITAL_OCEAN_AUTH_TOKEN')
+else:
+	api_token = input ('Enter your API token: ')
+
+
+if os.getenv('SSH_KEY'):
+	ssh_key = os.getenv('SSH_KEY')
+else:
+	ssh_key = input (' Enter your SSH Key path: ')
+
+# The api_url_base variable is the string that starts off every URL in the DigitalOcean API.
+api_url_base = 'https://api.digitalocean.com/v2/'
+# We need to set up the HTTP request headers the way the API docs describe.
+# Add these lines to the file to set up a dictionary containing your request headers:
+headers = {'Content-Type': 'application/json',
+		'Authorization': 'Bearer {0}'.format(api_token)}
+
+
+
+def get_all_droplets():
+	api_url = '{0}droplets'.format(api_url_base)
+	response = requests.get(api_url, headers=headers)
+
+	if response.status_code == 200:
+		return json.loads(response.content.decode('utf-8'))
+	else:
+		return None
+
+def del_droplet(dropletId):
+
+	droplet = digitalocean.Droplet(token=api_token,id=dropletId)
+
+	destroyed = droplet.destroy()
+	click.secho(">>> Success! Your droplet has been deleted.", fg='bright_green')
+	
+def del_domain(domain_name):
+
+	domain = digitalocean.Domain(token=api_token, name=domain_name)
+
+	destroyed = domain.destroy()
+
+	if destroyed == True:
+		click.secho(">>> Success! Your domain name '{}' has been deleted.".format(domain_name), fg='bright_green')
+	else:   
+		click.secho(' [!] Request failed!', fg='bright_red')
+
+
+def check_droplet(droplet):
+	my_droplets = get_all_droplets()
+	droplet_exists = ''
+	droplet_id: ''
+	
+	# Checks if the droplet already exists or not
+	for each_droplet, details in enumerate(my_droplets['droplets']):
+		for key, value in details.items():
+			x = details['name']
+			if droplet != x: continue
+			droplet_exists = x
+			return droplet_exists
+	else:
+		return None
+
+# Deletes all droplets
+def del_all_droplets():
+	my_droplets = get_all_droplets()
+
+	for each_droplet, details in enumerate(my_droplets['droplets']):
+		for key, value in details.items():
+			x = details['name']
+			dropletId = details['id']
+			del_droplet(dropletId)
+	else:
+		click.secho('All droplets have been deleted!', fg='bright_green')
+
+
+def create_droplet(droplet):
+
+	droplet = digitalocean.Droplet(token=api_token,
```

Comment:
See how this rendered weird in GitHub? It probably looks normal to you in your editor! THis just happens when you use tabs - Github is probably rendering each tab as 8 spaces wide, and you might be rendering them as 4 spaces wide or whatever, so to you this lines up neatly, but to other people it looks way off!!

Programmes have been arguing about tabs versus spaces for literally decades

https://thenewstack.io/spaces-vs-tabs-a-20-year-debate-and-now-this-what-the-hell-is-wrong-with-go/

https://kennethreilly.medium.com/tabs-vs-spaces-3c24defa7c9e

I say, I am so sick of this argument, that I just dont care any more :D if it looks good to you in your editor, then I dont care. If it's not lined up in your editor, then please go line it up.

I just thought I'de put this comment in to show you how stupid programmers can be, arguing over stuff that really doesn't matter :D